Source/WebCore/loader/cache/AllowedFonts.cpp

```diff
@@ -0,0 +1,814 @@
+        "gwJ7EEFiFqIKDXUiweNnoWlI9FDArGtMDVr3sFq58gc="_s,
+        "FlEBVC2LgznkUVVsGZ1RfNyIgzU3/R7+zOf1RpvOP+Y="_s,
+        "s7jCHt/mxeQC/cYHNm/Y0VlJplkU9YE0cz3GiSK8jWE="_s,
+        "f6Co0v+c9WNnrz2pJhF+jGSddB7CYBzgFimJhHcalUw="_s,
+        "hZSsfqI1BeLFCWZZxvZCxhDS7xJti6cYxCBnzMYzjDI="_s,
+        "WcGIwDIN+eM4qvcMy6OtTq5XFP9qSlsyP8rSk33hZN8="_s,
+        "mGDW8FOGoVDVNWr8JJpKZ/UWE3L+4sgMm4Ht3RMllwY="_s,
+        "ontWH6zp9U7py424W69njjB8G+Pmd0/KYG6X0xag0EA="_s,
+        "VLJGQZMntGDzevTy36pKycuQH2LqUXq5XAauhgXPUeY="_s,
+        "ljU6kYUWFmB88uf59hUZZ10WEZ3UO01Rvkkwso2iAzA="_s,
+        "bJFmac+SO08bLbXFEHyDtsogXnrQ3NhAslHmPwyNKKI="_s,
+        "Up0KezlEkpIiFVvKMnK6Goeswvqgmy7SanE4crf/h5Q="_s,
+        "jXmsIWS/PWBYPgxEcH951z+N47VeNl+YOckcy2tOXg8="_s,
+        "3uA7VZ7xUIL0hkAB1+zFYrL/TdLBp2EEfI/VKRmp7gQ="_s,
+        "14WAR75SQBRizwijKnP7jtwvCCT6tLw8ko+X1cEwFVA="_s,
+        "HCEME0w0ChEzmRpgWwqsmQzoUFxI/ON8kVBjQClk6ZQ="_s,
+        "iG/R5s6ZLt5RR0OoJvpx/v21dwR6KomcqV9vKPMwXQk="_s,
+        "p3/E2UhQ5amzlk0I9eLS+tXGY1umjTiyp+F89MVgvzc="_s,
+        "5HYWGFSBsOzWNtw1/EZzpKMysayWWF/B+c8NBVPawqA="_s,
+        "TcCPFfYc4WuXwMRYC9IzvgyepSFlMz/kYK9UBApcQlw="_s,
+        "lOG7wcKkHrxz+lJT/VYyVsADW01pGB5I+a755HShElE="_s,
+        "M6JS1jk8vW3r4KxRcinHqiWKDuaPwCU/i+anzui2Xuk="_s,
+        "Uj3HLKpCaHyUblqNfmtDiICiaIGWRGbPFCXOURTklwE="_s,
+        "aCNSdD2SKrdRVImxgEGyoD7vvpOQXM80C2vc3PXfY74="_s,
+        "VEAakg/RytnOL+sACpp0U0HdrrSAuCQ1YS+3k74gBv4="_s,
+        "wGkc3TsmPdNN85rSAsMeHvDd3HdtKPlpNrGzgWASFHc="_s,
+        "7Z2hTxTl3dXEcK1AO9v9Vef2KaUJjyDmCvKNOHqvFp0="_s,
+        "Tt+csKqcOW0oftpNrPEFlEmavyM5aR58vnRD2Gr65Jc="_s,
+        "pdjG8bPwgun7Pt99VzGoEDmvJ/5EYGDu2HRQ3jVdfd8="_s,
+        "i3ADysxd2P7zRlk8NNruecfR8JZO7Vftd9Ni6Xq3gvY="_s,
+        "IOSUZaQHdfd/sMO3lasmnJwc/mZefeU0J8k/Emo9zxQ="_s,
+        "9xL/UKwJf4Lk5VyH0ypUe4w61OH20/QTgLrClB3s2L4="_s,
+        "LK3tJCwEE5dhdC/gzaf2WS3xtmhoV1MsinwuJTa5duQ="_s,
+        "1Cpk3DSamAdei+Elh5Q/K9UgZai7GJYNfcc5C1NRF+A="_s,
+        "ziDtijIxF8inGP8d3G2ruZc3O1dajolvK/ArhGwILJ0="_s,
+        "4OPEryg0jXIfivYDWVwV0nOlbysDOS+aQTJV/lY19TY="_s,
+        "4nScskp3IIq92f2jXw0U8JGUjETSH5d8gEijsT5L7Ms="_s,
+        "Z5nJmeQicQ9A9wpgphOPw4EGImxE170bECP1u2W+/vk="_s,
+        "YSjdRP7ToEb/jYNdZ34Kg3xwxk3ByUS37f3gTrjYuHk="_s,
+        "GkbngM5b62UH1ir4sgqSszyPBC6HxhL0u/gzC/w1NBk="_s,
+        "qrlxreFjOrg2IiB0zq4KrYoILZAJCPJ0kbIh1ug5mMo="_s,
+        "XoEfCzLUiLmhg7d8/HrB70Sz6nqu0BToOXXf5ZfSIfY="_s,
+        "a1VZIONY+KJaQimItEhhXDO8zLT5MugzHOv8jipzf8c="_s,
+        "cbPOcmgPQYPSjbhrGEVCBR/VM7sRRpMyM+T2ogz5jLo="_s,
+        "oDdcBUoAQb1Y4qC/f6PffDkEv8T3kP0k4y/z7nD9Du8="_s,
+        "QE1ggxk+VpvFwox7G8DhPs6AxuD1pQrY6WM/SPPAkVU="_s,
+        "+YU60zfVI8CzX+esMGJopwNc4P92JHEO2LOcK4iyCjM="_s });
+
+    return allowedFontHashes;
+}
+
+static AtomString hashForFontData(const void* data, size_t size)
+{
+    auto cryptoDigest = PAL::CryptoDigest::create(PAL::CryptoDigest::Algorithm::SHA_256);
```

Comment:
Please add a comment above '`cryptoDigest->addBytes`':

`// FIXME(249414): Adopt Span in CryptoDigest`